frontend/app/(dashboard)/updates/company/[id]/page.tsx

```
@@ -2,11 +2,11 @@
 import { EnvelopeIcon } from "@heroicons/react/24/outline";
 import { useParams } from "next/navigation";
 import React from "react";
-import MainLayout from "@/components/layouts/Main";
 import MutationButton from "@/components/MutationButton";
 import RichText from "@/components/RichText";
 import { useCurrentCompany } from "@/global";
 import { trpc } from "@/trpc/client";
+import { DashboardHeader } from "@/components/DashboardHeader";
```

ESLint

[error] `@/components/DashboardHeader` import should occur before import of `@/components/MutationButton`

(import/order)